doc/manual/memory.rst

```diff
@@ -38,7 +38,7 @@ has been loaded with ``preload=True``. Preloading is also supported for :func:`m
 
 .. warning:: This comes with a caveat. When ``preload=False``, data rejection based on peak-to-peak thresholds is executed when the data is loaded from disk, *not* when the ``Epochs`` object is created.
 
-To explicitly reject artifacts with ``preload=False``, use the function :func:`mne.Epochs.drop_bad_epochs`.
+To explicitly reject artifacts with ``preload=False``, use the function :func:`mne.Epochs.drop_bad`.
```

Comment:
@dengemann was involved in the main thread where the names were brought up again as `drop_bad`, so I think so